08-06 19:44 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 1 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/novnc__websockify/requirements_licma_analysis/jwcrypto-0.7-py2.py3-none-any/jwcrypto/jwa.py
08-06 19:55 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 3 | cryptography | file: /media/ubuntu/DATA/repositories-python/novnc__websockify/requirements_licma_analysis/jwcrypto-0.7-py2.py3-none-any/jwcrypto/jwa.py
08-06 20:01 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/django__django/django/contrib/auth/hashers.py
08-06 20:03 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/encode__django-rest-framework/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 20:06 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/celery__celery/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 20:08 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/Miserlou__Zappa/requirements_licma_analysis/Django-3.0.3-py3-none-any/django/contrib/auth/hashers.py
08-06 20:10 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/wagtail__wagtail/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 20:12 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/mirumee__saleor/requirements_licma_analysis/Django-3.0.5-py3-none-any/django/contrib/auth/hashers.py
08-06 20:14 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/scrapinghub__portia/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 20:17 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/netbox-community__netbox/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 20:18 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/deis__deis/requirements_licma_analysis/Django-1.6.11-py2.py3-none-any/django/contrib/auth/hashers.py
08-06 20:20 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/MobSF__Mobile-Security-Framework-MobSF/requirements_licma_analysis/Django-3.0.6-py3-none-any/django/contrib/auth/hashers.py
08-06 20:24 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 4 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/MobSF__Mobile-Security-Framework-MobSF/requirements_licma_analysis/oscrypto-1.1.0-py2.py3-none-any/oscrypto/_asymmetric.py
08-06 20:27 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/readthedocs__readthedocs.org/requirements_licma_analysis/Django-2.2.12-py3-none-any/django/contrib/auth/hashers.py
08-06 20:29 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/taigaio__taiga-back/requirements_licma_analysis/Django-1.11.29-py2.py3-none-any/django/contrib/auth/hashers.py
08-06 20:31 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/mzucker__noteshrink/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 20:35 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 4 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/wbond__package_control/package_control/deps/oscrypto/_asymmetric.py
08-06 20:38 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/sshwsfc__xadmin/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 20:39 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/SpiderClub__weibospider/requirements_licma_analysis/Django-1.11.6-py2.py3-none-any/django/contrib/auth/hashers.py
08-06 20:41 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/django-oscar__django-oscar/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 20:43 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/cloudera__hue/desktop/core/ext-py/Django-1.11.29/django/contrib/auth/hashers.py
08-06 20:45 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/cloudera__hue/requirements_licma_analysis/Django-1.11.28-py2.py3-none-any/django/contrib/auth/hashers.py
08-06 20:47 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/cloudera__hue/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 20:49 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/cloudera__hue/requirements_licma_analysis/Django-1.10-py2.py3-none-any/django/contrib/auth/hashers.py
08-06 20:51 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/MVIG-SJTU__AlphaPose/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 20:53 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/django-crispy-forms__django-crispy-forms/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 20:55 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/offu__WeRoBot/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 20:57 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/eudicots__Cactus/requirements_licma_analysis/Django-1.6.11-py2.py3-none-any/django/contrib/auth/hashers.py
08-06 20:59 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/QingdaoU__OnlineJudge/requirements_licma_analysis/Django-2.1.7-py3-none-any/django/contrib/auth/hashers.py
08-06 21:01 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/samuelcolvin__pydantic/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 21:03 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/doccano__doccano/requirements_licma_analysis/Django-2.1.11-py3-none-any/django/contrib/auth/hashers.py
08-06 21:05 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/doccano__doccano/requirements_licma_analysis/Django-2.2.10-py3-none-any/django/contrib/auth/hashers.py
08-06 21:07 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/AKSHAYUBHAT__DeepVideoAnalytics/requirements_licma_analysis/Django-1.11.28-py2.py3-none-any/django/contrib/auth/hashers.py
08-06 21:09 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/healthchecks__healthchecks/requirements_licma_analysis/Django-3.0.4-py3-none-any/django/contrib/auth/hashers.py
08-06 21:11 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/liangliangyy__DjangoBlog/requirements_licma_analysis/Django-3.0.5-py3-none-any/django/contrib/auth/hashers.py
08-06 21:13 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/graphql-python__graphene-django/requirements_licma_analysis/Django-3.0.3-py3-none-any/django/contrib/auth/hashers.py
08-06 21:15 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/cobrateam__splinter/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 21:17 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/dyve__django-bootstrap3/requirements_licma_analysis/Django-3.0.5-py3-none-any/django/contrib/auth/hashers.py
08-06 21:19 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/awesto__django-shop/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 21:21 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/Greenwolf__social_mapper/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 21:23 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/zhoubear__open-paperless/requirements_licma_analysis/Django-1.10.7-py2.py3-none-any/django/contrib/auth/hashers.py
08-06 21:25 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/wechatpy__wechatpy/requirements_licma_analysis/Django-3.0.8-py3-none-any/django/contrib/auth/hashers.py
08-06 21:27 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/guohongze__adminset/requirements_licma_analysis/Django-1.11.28-py2.py3-none-any/django/contrib/auth/hashers.py
08-06 21:29 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 5 | M2Crypto | file: /media/ubuntu/DATA/repositories-python/marcgibbons__django-rest-swagger/requirements_licma_analysis/Django-2.1.3-py3-none-any/django/contrib/auth/hashers.py
08-06 21:33 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 2 | PyCrypto | file: /media/ubuntu/DATA/repositories-python/netbox-community__netbox/netbox/secrets/models.py
08-06 21:34 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 3 | PyCrypto | file: /media/ubuntu/DATA/repositories-python/netbox-community__netbox/netbox/secrets/models.py
08-06 21:51 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 3 | PyCrypto | file: /media/ubuntu/DATA/repositories-python/trustedsec__social-engineer-toolkit/requirements_licma_analysis/impacket-0.9.21/impacket-0.9.21/examples/dpapi.py
08-06 22:01 | ERROR | maximum recursion depth exceeded while calling a Python object | rule: PyQuery: Rule 3 | PyCrypto | file: /media/ubuntu/DATA/repositories-python/SecureAuthCorp__impacket/examples/dpapi.py
08-06 22:19 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 3 | PyCrypto | file: /media/ubuntu/DATA/repositories-python/byt3bl33d3r__CrackMapExec/requirements_licma_analysis/impacket-0.9.21/impacket-0.9.21/examples/dpapi.py
08-06 22:34 | ERROR | maximum recursion depth exceeded | rule: PyQuery: Rule 3 | PyCrypto | file: /media/ubuntu/DATA/repositories-python/guardicore__monkey/requirements_licma_analysis/impacket-0.9.21/impacket-0.9.21/examples/dpapi.py
